### PR TITLE
Generic Button style fixes

### PR DESCRIPTION
**Pull request by ehanna4**
Applied super basic style changes just to match the generic look of the other buttons

### CODE REVIEW

File: src/pages/workspaces/workspace/Notebooks.js [hunk 1]
```diff
@@ -319,7 +319,7 @@ const Notebooks = _.flow(
       div({
         style: {
           margin: '0 2.5rem 2.5rem 0', display: 'flex',
-          height: 250, width: 200, flexDirection: 'column',
+          height: 350, width: 200, flexDirection: 'column',
```
Comment (s-rubenstein) [inline src/pages/workspaces/workspace/Notebooks.js]: This seems like an unrelated change and doesn't look like what we want?

Comment (ehanna4) [reply src/pages/workspaces/workspace/Notebooks.js]: This is the height of the outer div - when it's only 250 the new button can't be very big. I don't know that this is 100% the way we want to go for the final choice (i.e. a hardcoded outer div height) but for now it allows the buttons to be big enough to see text.

Comment (ehanna4) [reply src/pages/workspaces/workspace/Notebooks.js]: ^ or that's how it seems to me and testing seems to fit this theory, but I could be misinterpreting what's controlling the button height style?

Comment (s-rubenstein) [reply src/pages/workspaces/workspace/Notebooks.js]: That will change existing buttons though, it seems to stretch them a lot taller.
Screen Shot 2020-09-03 at 1 13 41 PM

Comment (ehanna4) [reply src/pages/workspaces/workspace/Notebooks.js]: ah hmm I was trying to keep the pattern as similar to the existing as possible however now I'm wondering if this even should be using flex and maybe each button should just be setting their own height

Comment (ehanna4) [reply src/pages/workspaces/workspace/Notebooks.js]: or i could just add a max-height to the buttons for now?

**Review by s-rubenstein — APPROVED** [review]
LGTM
